.gitattributes

```diff
@@ -7,5 +7,6 @@
 /src/Symfony/Component/Translation/Bridge export-ignore
 /src/Symfony/Component/Emoji/Resources/data/* linguist-generated=true
 /src/Symfony/Component/Intl/Resources/data/*/* linguist-generated=true
+/src/Symfony/Component/TypeInfo/Tests/Fixtures/DummyWithUsesWindowsLineEndings.php text eol=crlf
```

Comment:
I don't think we need this if `src/Symfony/Component/TypeInfo/.gitattributes` already defines it